### PR TITLE
Splitting config file to themes

### PR DESCRIPTION
**Pull request by SimonbJohnson**
Leaving this in draft until the 3W config is ready to check it is working.

I pulled fields such as gender into a core as I wasn't sure if that would be used at some point across themes 

https://github.com/OCHA-DAP/hapi-pipelines/tree/splitting-config-file-to-themes/src/hapi/pipelines/configs

### CODE REVIEW

**Review by turnerm — COMMENTED** [review]
This looks really cool. I assume that ultimately we'd want to change the scraper to handle multiple configs - what do you think @mcarans @b-j-mills? But I think this is a great work-around in the meantime. 

Sorry in the meantime there's been a lot more population-related configuration added :sweat: But I think once you've pulled those in I would be fine with merging this.

@b-j-mills curious to know what you think!

**Comment by b-j-mills** [comment]
Agreed!  It looks great - I'm happy for you to pull in the new pop/3W changes and merge.  I defer to @mcarans about expanding facade I believe to accommodate multiple configs.

**Review by b-j-mills — APPROVED** [review]

**Comment by SimonbJohnson** [comment]
@b-j-mills @turnerm - I've merged main into this to solve conflicts. However when I run locally I get a keyerror as the HRPs have been removed from the config.

I also get this is in the current main branch.

Should I merge anyway?

```
raise KeyError(key)
    KeyError: 'HRPs'
```

File: src/hapi/pipelines/configs/population.yaml
Comment (turnerm) [inline src/hapi/pipelines/configs/population.yaml:944]: I think these are missing

File: src/hapi/pipelines/configs/population.yaml
Comment (turnerm) [inline src/hapi/pipelines/configs/population.yaml:907]: These lines should be removed

**Review by turnerm — CHANGES_REQUESTED** [review]
@SimonbJohnson are you sure you were running the latest version? I got a couple of different errors but they're solved if you make the changes below

File: src/hapi/pipelines/configs/population.yaml
Comment (SimonbJohnson) [reply src/hapi/pipelines/configs/population.yaml:907]: Updated to remove

File: src/hapi/pipelines/configs/population.yaml
Comment (SimonbJohnson) [reply src/hapi/pipelines/configs/population.yaml:944]: Updated to include

**Comment by SimonbJohnson** [comment]
Yes, problem was I hand't updated after the merge. All working now!

**Review by turnerm — APPROVED** [review]

**Comment by turnerm** [comment]
@SimonbJohnson can you just update the CHANGELOG? Then it should be good to merge!

**Pull request merged**
